### PR TITLE
Issue/24 implement resultsviewcontroller

### PR DESCRIPTION
**Pull request by saviotsui**
- [ ] Wait for https://github.com/teamnzs/ios-trivia-game/pull/40

resultsviewcontroller

### CODE REVIEW

File: ios-trivia-game/ViewControllers/QuestionViewController.swift [hunk 1]
```diff
@@ -18,12 +18,19 @@ class QuestionViewController: UIViewController {
         // Do any additional setup after loading the view.
         
         // @Nari or @Zhia: Feel free to remove the hardcoded values when you get to this
-        let questionId = 1932
+        let questionId = 1938
```
Comment (wassupnari) [inline ios-trivia-game/ViewControllers/QuestionViewController.swift:21]: Should we keep this for now? or should we remove it?

**Review by wassupnari — APPROVED** [review]
I have a minor question, but everything else looks good!

File: ios-trivia-game/ViewControllers/QuestionViewController.swift (hunk 1 above)
Comment (saviotsui) [reply ios-trivia-game/ViewControllers/QuestionViewController.swift:21]: this should be removed once questionviewcontroller PR is checked in.  For now, I guess it is okay?

Comment (wassupnari) [reply ios-trivia-game/ViewControllers/QuestionViewController.swift:21]: Okay! I'll merge this in!

**Pull request merged**